.setup/bin/sample_courses/models/course/course_utils.py

```diff
@@ -32,16 +33,16 @@ def __init__(self):
     def check_rotating(self, users) -> None:
         for gradeable in self.gradeables:
             for grading_rotating in gradeable.grading_rotating:
-                string = f"Invalid user_id {grading_rotating['user_id']} for rotating section for gradeable {gradeable.id}"
+                string = f"Invalid user_id {grading_rotating['user_id']} for "\
+                         f"rotating section for gradeable {gradeable.id}"
                 if grading_rotating['user_id'] not in users:
                     raise ValueError(string)
-                
+
     def getForumDataFromFile(self, filename) -> None:
         forum_path = os.path.join(SETUP_DATA_PATH, "forum")
         forum_data = []
         for line in open(os.path.join(forum_path, filename)):
-            l = [x.replace("\\n", "\n").strip() for x in line.split("|")]
-            if(len(line) > 1):
-                forum_data.append(l)
+            parsed_line = [x.replace("\\n", "\n").strip() for x in line.split("|")]
+            if (len(line) > 1):
```

Comment:
The parentheses are redundant here:
```suggestion
            if len(line) > 1:
```